doc/config_rsmexplain.rst.inc

```diff
@@ -0,0 +1,66 @@
+.. _config_file_rsmexplain:
+
+Experiment configuration file
+"""""""""""""""""""""""""""""
+
+This is a file in ``.json`` format that provides overall configuration options for an ``rsmexplain`` experiment. Here's an `example configuration file <https://github.com/EducationalTestingService/rsmtool/blob/main/examples/rsmexplain/config_rsmexplain.json>`__ for ``rsmexplain``.
+
+.. note:: To make it easy to get started with  ``rsmexplain``, we provide a way to **automatically generate** configurations file both interactively as well as non-interactively. Novice users will find interactive generation more helpful while more advanced users will prefer non-interactive generation. See :ref:`this page <autogen_configuration>` for more details.
+
+Next, we describe all of the ``rsmexplain`` configuration fields in detail. There are four required fields and the rest are all optional. We first describe the required fields and then the optional ones (sorted alphabetically).
+
+background_data
+~~~~~~~~~~~~~~~
+The path to the background data feature file one of the :ref:`supported formats <input_file_format>`. Each row should correspond to a single response and contain the numeric feature values extracted for this response. In addition, there should be a column containing a unique identifier (ID) for each response. This path can be absolute or relative to the location of config file. It must contain at least 300 responses to ensure meaningful explanations.
+
+explainable_data
```

Comment:
I see what you mean. `explainable_data` sounds to me like there is some requirement on the data, that it will be "explainable". but if this is the naming used by shap, that's fine.

Reply:
I don't think SHAP has the convention to call this `explainable`. I think this is a name we just chose. I agree with @tamarl08  that `explainable` does imply something other than "this is the data we want to explain". How about `explained_data`?

Reply:
yes `explained_data` is better.  how about `explain_data`?

Reply:
It's definitely shorter but it kind of breaks the cognitive alignment with `background_data by using an imperative instead of an adjective as the first word? May be I am just overthinking it?

Reply:
Overthinking here too, explained_data suggests that it has already been explained.
`explanation_data`?

(but really, all the previous suggestions are fine)

Reply:
Yeah, I agree. `explain_data` sounds the best out of all the options to me. @damien2012eng can you please rename `explainable_data` to `explain_data` in all the code and documentation?